src/core/RandomVariables.jl

```diff
@@ -67,9 +67,21 @@ cuid(vn::VarName) = (vn.csym, vn.sym, vn.indexing)    # the uid which is only av
 
 copybyindex(vn::VarName, indexing::String) = VarName(vn.csym, vn.sym, indexing, vn.counter)
 
-###########
-# VarInfo #
-###########
+function is_inside(vn::VarName, space::Set)::Bool
+    if vn.sym in space
+        return true
+    else
```

Comment:
Let's add some comments to this old function. E.g. here we can say "1. collect expressions from `space`. 2. convert them to strings (`strs`). 3. convert `vn` to string (`vn_str`). 4. check if `vn_str` is in the `strs`.

Also maybe rename `strs` to `expr_strs`?

Reply:
Sure.